nbody_gravitational/NBodyGravitational.jl

```diff
@@ -0,0 +1,173 @@
+module NBodyGravitational
+
+using DiffEqBase, DifferentialEquations, StaticArrays, Plots, StaticArrays
+
+
+grav_const = 6.673e-11
```

Comment:
Are there units where this is more numerically okay? This kind of a constant may cause cancellation

Reply:
Sure, but this one is in SI units as a default value. We can set the desired value in constructor of NBodyGravProblem. If  I change artificially `grav_const` to a greater value during computation, the multiplication of stellar masses with this constant will give rather big number - it may also cause problems.

Reply:
I removed this const - it is unnecessary here (see further)